code/datums/diseases/parasitic_infection.dm

```diff
@@ -13,18 +13,28 @@
 	spread_flags = DISEASE_SPREAD_NON_CONTAGIOUS
 	required_organs = list(/obj/item/organ/liver)
 	bypasses_immunity = TRUE
+	var/obj/item/organ/liver/affected_liver
 
 
 /datum/disease/parasite/stage_act()
 	. = ..()
 	if(!.)
 		return
-
-	var/obj/item/organ/liver/affected_liver = affected_mob.getorgan(/obj/item/organ/liver)
```

Comment:
This was the only code that existed in this previously. Every stage act it would check for a liver but it would not check if a liver ever existed.